runtime/lua/vim/lsp/completion.lua

```diff
@@ -31,9 +31,9 @@ local buf_handles = {}
 --- @nodoc
 --- @class vim.lsp.completion.Context
 local Context = {
-  cursor = nil, --- @type [integer, integer]?
+  cursor = nil,            --- @type [integer, integer]?
```

Comment:
Could we please remove the formatting changes in this PR?

Reply:
i will withdraw this pr. because i found too-frequent trigger autocompletion will bring more bug.

`clangd`  fails to set `isIncomplete` to `true` in time when calling lsp-snippets:
[录屏 12-26-2024 10:22:06 AM.webm](https://github.com/user-attachments/assets/5d2c96b8-2a3d-4fe8-9ca3-2304dd85fd06)

`lua_ls` will still have the bug of `Using a Blob as a String`  when inserting the lsp-snippets:

[录屏 12-26-2024 10:24:44 AM.webm](https://github.com/user-attachments/assets/61c1f7a4-316b-4f4d-9521-9af6f8e1ad8d)


it needs more considerations. Anyway, both `"TextChangedI", "TextChangedP", "InsertCharPre"` are  not very suitable for this feature (in my opinion).

Reply:
I now use a very dirty method, directly blocking lsp-snippet to make it work normally.

Reply:
It would still be nice to have reproduction steps for your issue. If there's a bug we ought to fix it.
But a PR that fixes a bug shouldn't mix other features into it (re-trigger on backspace)

Reply:
sorry for it. Above bugs do not exist in the master; maybe i need more coding and debuging for re-trigger on backspace/alphabet.

Reply:
can you close this pr?